WebARKit/WebARKitTrackers/WebARKitOpticalTracking/WebARKitTracker.cpp

```diff
@@ -119,6 +116,10 @@ class WebARKitTracker::WebARKitTrackerImpl {
             cv::Mat colorFrame(frameRows, frameCols, CV_8UC4, frameData);
```

Comment:
We could create an uitlity function to avoid repetition here and in initTracker, enhancing the readability of the code as well.

Reply:
done with https://github.com/webarkit/WebARKitLib/pull/23/commits/022b55951b766496ed9441ed0c8707d176ebb21b